flint/docs/index.md

```diff
@@ -171,8 +171,15 @@ In the index mapping, the `_meta` and `properties`field stores meta and schema i
 
 #### Configurations
 
-- `spark.flint.indexstore.location`: default is localhost
-- `spark.flint.indexstore.port`: default is 9200
+- `spark.datasource.flint.location`: default is localhost.
+- `spark.datasource.flint.port`: default is 9200.
+- `spark.datasource.flint.scheme`: default is http. valid values [http, https]
+- `spark.datasource.flint.auth`: default is false. valid values [false, sigv4]
+- `spark.datasource.flint.region`: default is us-west-2. only been used when auth=sigv4
+- `spark.datasource.flint.write.id_name`: no default value.
+- `spark.datasource.flint.write.batch_size`: default value is 1000.
+- `spark.datasource.flint.write.refresh_policy`: default value is 1000. valid values [NONE(false), IMMEDIATE(true), WAIT_UNTIL(wait_for)]
```

Comment:
default value is false?

Reply:
Could you update the TODO at line 75 with example too?

Reply:
done.